From a4daefe3790ed0c5863564dbbce129af2d954f0d Mon Sep 17 00:00:00 2001
From: Bin Wang <wangbin224@huawei.com>
Date: Wed, 26 Jan 2022 20:00:10 +0800
Subject: [PATCH] x86/kdump: add log before booting crash kernel
Patch-mainline: Not yet, from openEuler
References: bsn#22
openEuler-commit: a4daefe3790ed0c5863564dbbce129af2d954f0d
Modified-by-SEL: No


from euler inclusion
category: feature
bugzilla: https://gitee.com/openeuler/kernel/issues/I4S9W4?from=project-issue
CVE: NA

---------------------------

Just like arm64, print "Bye!" before booting crash kernel.

Signed-off-by: Bin Wang <wangbin224@huawei.com>
Reviewed-by: luo chunsheng <luochunsheng@huawei.com>
Reviewed-by: Yang Jihong <yangjihong1@huawei.com>
Signed-off-by: Zheng Zengkai <zhengzengkai@huawei.com>
Signed-off-by: Guoqing Jiang <guoqing.jiang@suse.com>
---
 arch/x86/kernel/machine_kexec_64.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/arch/x86/kernel/machine_kexec_64.c b/arch/x86/kernel/machine_kexec_64.c
index 055c18a6f7bf..ab5163f6ae00 100644
--- a/arch/x86/kernel/machine_kexec_64.c
+++ b/arch/x86/kernel/machine_kexec_64.c
@@ -364,6 +364,8 @@ void machine_kexec(struct kimage *image)
 		page_list[PA_SWAP_PAGE] = (page_to_pfn(image->swap_page)
 						<< PAGE_SHIFT);
 
+	pr_info("Bye!\n");
+
 	/*
 	 * The segment registers are funny things, they have both a
 	 * visible and an invisible part.  Whenever the visible part is
-- 
2.26.2

